guides/common/modules/proc_excluding-hosts-from-receiving-proxied-requests-by-using-cli.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="excluding-hosts-from-receiving-proxied-requests-by-using-cli"]
+= Excluding hosts from receiving proxied requests by using CLI
+
+If you use an HTTP proxy for all {Project} HTTP or HTTPS requests, you can prevent certain hosts from communicating through the HTTP proxy.
```

Comment:
```suggestion
If you use an HTTP proxy for all HTTP or HTTPS requests to {Project}, you can prevent certain hosts from communicating through the HTTP proxy.
```
Should make the direction of the requests clearer (from/to).
Unless it's both directions, then you could keep it as is.

Reply:
I don't know. My feeling is "from". Can someone please confirm/clarify?

no change yet.

Reply:
Discussed internally; "from {Project}" seems the way to go.